File: hw/pantoja/u2/hw18ZooInPythonAndJavaScript/ZooInPython/ec_edu_espe_zoo_model/Tiger.py
from datetime import datetime
from Feline import Feline

class Tiger(Feline):
    def __init__(self, id, description, date_onborn, gender, number_of_bones, cage):
        super().__init__(id, description, date_onborn, gender, number_of_bones, cage)

    def feed(self, food):
        print("Feeding to a lion")

File: hw/villagomez/u2/hw18ZooJS/ec.edu.espe.zoo.model/Tiger.js
const Feline = require('./Feline');
const Cage = require('./Cage'); 

class Tiger extends Feline {

    constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

    feed(food) {
        throw new Error('Not supported yet.');
    }
    
}

module.exports = Tiger;